packages/react-core/src/components/Radio/examples/RadioControlled.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Radio } from '@patternfly/react-core';
+
+export const RadioControlled: React.FunctionComponent = () => {
+  const [check1, setCheck1] = React.useState(false);
+
+  function handleChange(_: boolean) {
```

Comment:
I don't even think this needs the optional `checked` param. And I think usually we write our helper functions in the following functional format:
```
const handleChange = () => {...}
```

Reply:
Should be fixed by the new commit